.github/workflows/runTests.yaml

```diff
@@ -73,6 +75,68 @@ jobs:
         name: Playwright test results ${{ matrix.project }}
         path: test-results
 
+  runCrosspostTests:
```

Comment:
Does this need to be a separate job? There are quite a lot of jobs piling up now

Reply:
It doesn't _need_ to be, but I feel like it's better for now just because it makes running the tests more complicated and I don't want to raise the barrier for entry for adding more tests - I want it to get easier.

I think this is ~fine as a separate test suite since it's mainly a regression test to catch bugs in CI, but I agree this isn't exactly ideal longterm.

The main issue is that playwright seems to be a bit unreliable with the `webservers` setting that we use to start the servers and the databases. This is also related to your issues getting it to work with `--ui` - the easiest fix is probably to temporarily disable the webservers in `playwright.config.ts`, and then to start them manually in separate terminals.

At some point we should probably spend some time tidying this up, but it doesn't seem super high prio right now.